test/scripts/integration/test-integration.py

```
@@ -0,0 +1,1348 @@
+        r = self._get("/olla/anthropic/v1/models")
+        ok = False
+        detail = ""
+        if r and r.status_code == 200:
+            try:
+                r.json()
+                ok = True
+            except Exception:
+                detail = "invalid JSON"
+        elif r and r.status_code == 404:
+            detail = "endpoint not configured (404)"
+            self.pcolor(YELLOW, f"  [SKIP] Anthropic translator not configured, skipping phase")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)